app/uk/gov/hmrc/agentclientrelationships/connectors/UsersGroupsSearchConnector.scala

```diff
@@ -49,6 +63,17 @@ class UsersGroupsSearchConnector @Inject()(
     extends HttpAPIMonitor {
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
+  def getFirstGroupAdminUser(groupId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[UserDetails] = {
+    val url = new URL(baseUrl, s"/users-groups-search/groups/$groupId/users")
+    monitor(s"ConsumedAPI-UGS-getGroupUsers-GET") {
+      httpGet
+        .GET[Seq[UserDetails]](url.toString)
+        .map(_.find(_.credentialRole.exists(_ == "Admin")).getOrElse(throw AdminNotFound("NO_ADMIN_USER")))
+    } recoverWith {
+      case _: NotFoundException => Future failed RelationshipNotFound("UNKNOWN_GROUP")
```

Comment:
again we are not looking for relation here, so we cant throw `RelationshipNotFound ` ?

Reply:
Yeah, I think I copied this from another method, and I also thought the same... will fix